playback/src/decoder/passthrough_decoder.rs

```diff
@@ -32,19 +32,19 @@ where
     Ok(pck.data.into_boxed_slice())
 }
 
-pub struct PassthroughDecoder<R: Read + Seek> {
+pub struct PassthroughDecoder<'a, R: Read + Seek> {
```

Comment:
Since we're only writing owned data, we shouldn't need this lifetime here and can instead specify `'static` for `PacketWriter`.

Reply:
Thanks this worked.